tests/models/gemma/test_modeling_gemma.py

```diff
@@ -607,8 +607,8 @@ def test_model_2b_eager(self):
         # considering differences in hardware processing and potential deviations in generated text.
         EXPECTED_TEXTS = {
             7: [
-                "Hello I am doing a project on the 1990s and I am looking for some information on the ",
-                "Hi today I am going to share with you a very easy and simple recipe of <strong><em>Kaju Kat",
+                "Hello I am doing a project on the 1990s and I need to know what the most popular music",
+                "Hi today I am going to share with you a very easy and simple recipe of <strong><em>Khichdi",
```

Comment:
This is in `def test_model_2b_eager(self):` which is always failing (previously token issue and assertion failed after a fix for it)